### DISCUSSION TITLE
Question: is there any way to design the ui using a tool like Glade

### DISCUSSION THREAD

**Discussion by fedpo2** [gircore/gir.core#1081] [Q&A] [answered]
i searched for any reference in the API documentation and couldn't find anything related to it
![image](https://github.com/gircore/gir.core/assets/61921832/9c2670f0-df7a-4325-a1a0-949dfc517e61)

pd: english is not my first language.

**Comment by badcel** [comment, marked as answer]
Hi @fedpo2 ,

i converted your issue into a discussion as it is a non technical question.

To answer it, yes there is Cambalache: https://flathub.org/apps/ar.xjuan.Cambalache

Additionally there is [blueprint](https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/) as an alternative to xml but as far as I know there is no graphical tool for it.

**Reply by fedpo2** [reply]
thanks

**Reply by Daniel15** [reply]
It's not exactly a graphical too, but you can use [GNOME Workbench](https://apps.gnome.org/Workbench/) for Blueprint. You have to type the Blueprint file, but it shows a preview, and comes with a lot of examples.
